source/_data/glossary.yml

```
@@ -32,12 +32,12 @@
   description: "[Groups](/integrations/group/) are a way to organize your entities into a single unit."
 - topic: hass
   description: "HASS or [hass](/docs/tools/hass/) is often used as an abbreviation for Home Assistant. It is also the command-line tool."
-- topic: Hass.io
-  description: "[Hass.io](/hassio/) is a full UI managed home automation ecosystem that runs Home Assistant, the Hass.io Supervisor and add-ons. It comes pre-installed on HassOS, but can be installed on any Linux system. It leverages Docker, which is managed by the Hass.io Supervisor."
-- topic: Hass.io Supervisor
-  description: "The Hass.io Supervisor is a program that manages a Hass.io installation, taking care of installing and updating Home Assistant, add-ons, itself and, if used, updating the HassOS operating system."
-- topic: HassOS
-  description: "HassOS, the Home Assistant Operating System, is an embedded, minimalistic, operating system designed to run the Hass.io ecosystem on single board computers (like the Raspberry Pi) or Virtual Machines. The Hass.io Supervisor can keep it up to date, removing the need for you to manage an operating system."
+- topic: Home Assistant Supervised (Deprecated Hass.io)
+  description: "[Home Assistant](/hassio/) is a full UI managed home automation ecosystem that runs Home Assistant, the Home Assistant Supervisor and add-ons. It comes pre-installed on Home Assistant OS, but can be installed on any Linux system. It leverages Docker, which is managed by the Home Assistant Supervisor."
+- topic: Home Assistant Supervisor
+  description: "The Home Assistant Supervisor is a program that manages a Home Assistant installation, taking care of installing and updating Home Assistant, add-ons, itself and, if used, updating the HassOS operating system."
+- topic: Home Assistant OS
```

Let's not abbreviate, applies to other content in this PR as well.

```suggestion
- topic: Home Assistant Operating System
```

---

Agreed